src/main/java/org/skife/jdbi/v2/BasicHandle.java

```diff
@@ -487,4 +490,20 @@ public void registerContainerFactory(ContainerFactory<?> factory)
     {
         this.containerFactoryRegistry.register(factory);
     }
+
+    @Override
+    public void setSqlObjectContext(Class<?> sqlObjectType, Method sqlObjectMethod) {
```

Comment:
Actually it's not so bad, I'm halfway done with it. Let me run with this and see where it gets us.